pvacseq/lib/pvacseq_utils.py

```diff
@@ -19,3 +23,50 @@ def iedb_to_prediction_method_lookup_dict():
 def iedb_prediction_methods():
     prediction_method_lookup_dict = iedb_to_prediction_method_lookup_dict()
     return sorted(prediction_method_lookup_dict.keys())
+
+def valid_allele_names_for_method(iedb_method):
+    #Ultimately we probably want this method to call out to IEDB but their command is currently broken
+    #curl --data "method=ann&species=human" http://tools-api.iedb.org/tools_api/mhci/
+    base_dir               = os.path.abspath(os.path.join(os.path.dirname(os.path.realpath(__file__)), '..'))
+    iedb_alleles_dir       = os.path.join(base_dir, 'iedb_alleles')
+    iedb_alleles_file_name = os.path.join(iedb_alleles_dir, "%s.tsv" % iedb_method)
+    alleles = set()
+    with open(iedb_alleles_file_name) as iedb_alleles_file:
```

Comment:
I just pushed a new commit to store the file contents. That was surprisingly easier than I thought it would be. I wonder if we need to do something similar for `valid_lengths_for_allele_and_method`, possibly combining it with the new `valid_allele_names_for_method_dict` so that the files only ever need to be read once.